source/includes/complete-examples.rst

```diff
@@ -1,6 +1,7 @@
-.. cta-banner::
-   :url: https://github.com/mongodb/docs-atlas-architecture/blob/main/source/includes/examples/tf-staging-prod-complete/
-   :icon: Code
+.. tip::
 
-   See Terraform examples to enforce our Staging/Prod recommendations across all pillars in one place `in Github <https://github.com/mongodb/docs-atlas-architecture/blob/main/source/includes/examples/tf-staging-prod-complete/>`__.
+   For Terraform examples that enforce our recommendations across all
+   pillars, see one of the following examples in Github:
 
+   - `Staging/Prod <https://github.com/mongodb/docs-atlas-architecture/blob/main/source/includes/examples/tf-staging-prod-complete/>`__
+   - `Dev/Test <https://github.com/mongodb/docs-atlas-architecture/blob/main/source/includes/examples/tf-dev-test-complete/>`__
```

Comment:
Note: These links will not work until this PR and https://github.com/mongodb/docs-atlas-architecture/pull/173 are merged and deployed

Reply:
can you update the ticket you filed about updating links after a monorepo migration to include these as well? thanks